install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -70,8 +87,7 @@ node16.example.com openshift_schedulable=True openshift_node_labels="{'role': 'i
 or infrastructure applications to be scheduled on them. It is up to the
 administrator to configure how applications will be constrained.
 
-. Run the Advanced Installer without deploying Logging or Metrics. This can be
-either as part of an initial {product-title} installation:
+. Run the Advanced Installer. This can be either as part of an initial {product-title} installation:
```

Comment:
s/Advanced Installer/advanced installer.

I'd separate the two examples with an ul, like:
** For your initial {product-title} installation, run the following commands:
+
commands
** To update an existing installation, run the following command:
+ 
command

Reply:
Advanced Installer is the name of the tool, hence capitalized.

Makes sense to split up the commands, I'll do that.